controllers/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -167,6 +167,7 @@ func (r *Reconciler) ensureUserAccount(logger logr.Logger, murAccount toolchainv
 			if !isStatusPresent {
 				counter.IncrementUserAccountCount(logger, murAccount.TargetCluster)
 			}
+
```

Comment:
Is there some UserAccount Status we should be checking at all?

Reply:
I beleive it is not needed, as the SynchStatus and SyncSpec take care of it.